src/components/filterModal/FilterModal.vue

```diff
@@ -164,7 +164,7 @@ export default {
         importStat() {
             this.$store.commit('setCurrentConditions', ['import', this.importedStat]);
             this.$store.dispatch('getListByConditions').then(() => {
-                this.$store.commit('setCurrentConditions', ['enum', this.importedStat, null, ['True']]);
+                this.$store.commit('setCurrentConditions', ['enum', this.importedStat, null, ['Proband']]);

```

Comment:
You don't have to hardcode value ('Proband'). I suppose the idea was only to import value on submit modal window, not to add it to conditions